examples/sveltekit-oidc-cookies-auth/package.json

```
@@ -0,0 +1,58 @@
+{
+	"name": "wundergraph-sveltekit-oidc-cookies-auth",
+	"version": "0.0.1",
+	"private": true,
+	"scripts": {
+		"start": "npm run start:services && npm run wait-on:services && npm run db:push && run-p dev wundergraph open",
+		"start:services": "docker-compose up -d",
+		"wait-on:services": "wait-on -d 5000 tcp:54321",
+		"wundergraph": "wunderctl up --debug",
+		"open": "wait-on -d 500 http://localhost:9991 && open-cli http://localhost:5173",
+		"build:wundergraph": "wunderctl generate --debug",
+		"build:sveltekit": "vite build",
+		"build": "npm run build:wundergraph && npm run build:sveltekit",
+		"dev": "vite dev",
+		"cleanup": "docker-compose down -v",
+		"wunderctl": "wunderctl",
+		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+		"preview": "vite preview",
+		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+		"test:unit": "vitest",
+		"lint": "prettier --plugin-search-dir . --check . && eslint .",
+		"format": "prettier --plugin-search-dir . --write .",
+		"db:push": "prisma db push",
+		"db:pull": "prisma db pull"
+	},
+	"devDependencies": {
+		"@sveltejs/adapter-auto": "^2.0.0",
+		"@sveltejs/kit": "^1.5.0",
+		"@typescript-eslint/eslint-plugin": "^5.45.0",
+		"@typescript-eslint/parser": "^5.45.0",
+		"autoprefixer": "^10.4.14",
+		"eslint": "^8.28.0",
+		"eslint-config-prettier": "^8.5.0",
+		"eslint-plugin-svelte3": "^4.0.0",
+		"graphql": "^16.6.0",
+		"npm-run-all": "^4.1.5",
+		"open-cli": "^7.2.0",
+		"postcss": "^8.4.21",
+		"prettier": "^2.8.0",
+		"prettier-plugin-svelte": "^2.8.1",
+		"prisma": "^4.12.0",
+		"svelte": "^3.54.0",
+		"svelte-check": "^3.0.1",
+		"tailwindcss": "^3.3.0",
+		"tslib": "^2.4.1",
+		"typescript": "^5.0.0",
+		"vite": "^4.2.0",
+		"vitest": "^0.25.3",
+		"wait-on": "^7.0.1"
+	},
+	"type": "module",
+	"dependencies": {
+		"@prisma/client": "^4.12.0",
+		"@tanstack/svelte-query": "^4.27.0",
+		"@wundergraph/sdk": "^0.143.0",
+		"@wundergraph/svelte-query": "^0.1.0"
```

Changing this svelte query package version to latest should be enough to merge this PR :D 
```suggestion
		"@wundergraph/svelte-query": "^0.2.2"
```